kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -89,6 +97,11 @@ window.LoginView = BaseView.extend({
     handle_login: function(response) {
         if (response.status == 200) {
             this.trigger("login_success");
```

Comment:
Does this set up the potential for a race condition? Maybe "login_success" should only be triggered if a redirect doesn't occur. Otherwise it's possible someone might come to expect a side effect from the "login_success" event that occurs before the page reloads, which isn't a safe expectation.